website/contact/index.html

```diff
@@ -51,7 +51,7 @@ <h3>{{ _('<a href="https://support.mozilla.org/questions/new/thunderbird">Ask a
         </section>
         <section>
           <h3>{{ _('<a href="irc://irc.mozilla.org/thunderbird">Live Chat on IRC</a>') }}</h3>
-          <p>{{ _('You can chat with other users and members of the Thunderbird team on the <a href="irc://irc.mozilla.org/thunderbird">#thunderbird channel on Mozilla’s IRC</a>. You can use IRC via Thunderbird in by following ') }}</p>
+          <p>{{ _('You can chat with other users and members of the Thunderbird team on the <a href="irc://irc.mozilla.org/thunderbird">#thunderbird channel on Mozilla’s IRC</a>. You can set up IRC in Thunderbird by following the <a href="https://support.mozilla.org/kb/instant-messaging-and-chat#w_configuring-chat">instructions on SUMO</a>.') }}</p>
```

Comment:
If only Thunderbird actually handled irc:// links. 😢

Reply:
@clokep you think I should remove it?

Reply:
No, it's fine.